wiki/Infrastructure-Efficiency.md

```diff
@@ -0,0 +1,41 @@
+# Infrastructure Efficiency
+
+- [Infrastructure Efficiency](#infrastructure-efficiency)
+  - [GitHub Actions](#github-actions)
+  - [Vercel](#vercel)
+    - [Website](#website)
+    - [Showcase](#showcase)
+
+---
+
+To conserve resources and improve efficiency we have steps in place to limit when tests are run and preview apps are deployed based on the following matrix:
+
+| Project Changed | Showcase Preview | Website Preview | Components CI | Ember Flight Icons CI | Website CI |
+| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- | 
+| Components  | ✅  | ❌  | ✅  | ❌  | ❌  |
+| Website  | ❌  | ✅  | ❌  | ❌  | ✅  |
+| Ember Flight Icons  | ✅  |  ❌  | ❌  | ✅  | ❌  |
+| Flight Icons  | ✅  | ✅  | ✅  | ❌  | ✅  | 
+
+## GitHub Actions
+
+We leverage conditional jobs across our CI suite to limit CI runs. Where needed we pass in a list of strings of directories we want to trigger CI for if they change, and pass them into [a bash script](https://github.com/hashicorp/design-system/blob/main/.github/scripts/filter_changed_files.sh). If the bash script detects changes in the referenced directories the rest of the CI process will proceed, otherwise the job will be marked as Skipped.
```

Comment:
```suggestion
We leverage conditional jobs across our CI suite to limit CI runs. Where needed, we pass in a list of strings of directories we want to trigger CI for if they change, and pass them into [a bash script](https://github.com/hashicorp/design-system/blob/main/.github/scripts/filter_changed_files.sh). If the bash script detects changes in the referenced directories, the rest of the CI process will proceed; otherwise, the job will be marked as “Skipped”.
```